### ISSUE TITLE
Check if service is exist and alarm if not working

### ISSUE THREAD

**Issue by maksymsereda** [netdata/netdata#5510]
##### Question summary
I am using NetData to monitor few servers with some services. In `app_groups.conf` named services witch I want to monitor
```
celery: celery*
apache: apache*
redis: redis*
postgres: postgres*
clamd: clamd* 
freshclam: freshclam*
supervisor: supervisor*
```
and in `/usr/lib/netdata/conf.d/health.d/*.conf` created conf files for each one as a sample
```
   alarm: celery_processes
      on: apps.processes
    calc: $celery
   every: 10s
    crit: $this = 0
   units: processes
    info: celery currently running processes
```
and it is working like a charm in that server. But I made a script with all conf files from `/usr/lib/netdata/conf.d/health.d/` and after install on another server everything is working but I have an alarm that celery (and other services) not working. This is a new empty server so there are no any services.

##### OS / Environment
Debian 9.6

##### Expected results
Is there a way how to check if service is installed and after that send an alarm that it is not working?

**Comment by ilyam8** [comment]
Hi @maksymsereda 	

try this:

```bash
   alarm: celery_processes
      on: apps.processes
    lookup: min -10s match-names of celery
   every: 10s
    crit: $this != nan AND $this = 0
   units: processes
    info: celery currently running processes
```

**Comment by maksymsereda** [comment]
Hi @ilyam8 

this is almost doing the trick. If I firstly install the service conf is working well, but when I deleted it (ex. celery) there is still an alarm that celery is not working. Is it a normal behavior?

Same with apache with standart config
```
# make sure apache is running

template: apache_last_collected_secs
      on: apache.requests
    calc: $now - $last_collected_t
   units: seconds ago
   every: 10s
    warn: $this > (($status >= $WARNING)  ? ($update_every) : ( 5 * $update_every))
    crit: $this > (($status == $CRITICAL) ? ($update_every) : (60 * $update_every))
   delay: down 5m multiplier 1.5 max 1h
    info: number of seconds since the last successful data collection
      to: webmaster
```

**Comment by ilyam8** [comment]
hm, i think apps_plugin will continue to plot zeros after proces dies.

@vlvkobal confirm or deny

**Comment by maksymsereda** [comment]
@ilyam8 
`systemctl restart netdata` helped, but maybe there is another way

**Comment by ilyam8** [comment]
@maksymsereda 	i think this is a bug, i'll open an issue, will see

**Comment by ilyam8** [comment]
@maksymsereda we can't distinguish if the process is not running or not installed, in both cases number of threads for defined group is zerop

**Comment by ilyam8** [comment]
related #5547

fixed in #5540
